client/webserver/site/src/css/main.scss

```diff
@@ -427,10 +427,9 @@ hr.dashed {
   background-color: #7775;
 }
 
-div.form-closer,
-div.corner-bttn {
```

Comment:
`corner-bttn` is clearly a relic, but for the future, it's helpful to comment on the PR to state that it's unused.

It does seem to have been orphaned in https://github.com/decred/dcrdex/commit/ebfcff744ad90754aba651c0a5d0b998543771a3#diff-9b6521c407894cf405fa6e27af8fb8ae95c2e629d563180d39c05d0cc0563f17L66